api.php

```diff
@@ -1,16 +1,10 @@
 <?php
 
 /**
- * OpenMage
- *
- * This source file is subject to the Open Software License (OSL 3.0)
- * that is bundled with this package in the file LICENSE.txt.
- * It is also available at https://opensource.org/license/osl-3-0-php
- *
  * @category   Mage
  * @package    Mage
  * @copyright  Copyright (c) 2006-2020 Magento, Inc. (https://www.magento.com)
- * @copyright  Copyright (c) 2021-2024 The OpenMage Contributors (https://www.openmage.org)
+ * @copyright  Copyright (c) 2021-2025 The OpenMage Contributors (https://www.openmage.org)
```

Comment:
these lines (bumping the year) should be avoided in the context of this PR

Reply:
The comments I make are for the good of this project. Also, you blocked me and unblocked me so, if you don't want my comments why did you unblock me.